### PR TITLE
Close sentry client at logger close

### PR DESCRIPTION
**Pull request by tagomoris**
Related with https://github.com/treasure-data/bigdam/issues/37

### CODE REVIEW

**Comment by tagomoris** [comment]
@komamitsu Could you review this change?

File: src/main/java/com/treasuredata/bigdam/log/Log.java [hunk 1]
```diff
@@ -95,9 +94,9 @@ public static Logger defaultLoggerGetter(final Class<?> clazz)
         return LoggerFactory.getLogger(clazz);
     }
 
-    public static SentryClient defaultSentryClientGetter()
+    public static SentryClient defaultSentryClientGetter(final String dsn)
     {
-        return SentryClientFactory.sentryClient();
+        return Sentry.init(dsn);
```
Comment (komamitsu) [inline src/main/java/com/treasuredata/bigdam/log/Log.java:99]: Is it better to keep this instance as singleton at the first time and to return it when this method is called next time?

Comment (komamitsu) [reply src/main/java/com/treasuredata/bigdam/log/Log.java:99]: FYI : According source code in Sentry, you can write like `SentryClientFactory.sentryClient(dsn, null)`

```
    public static SentryClient init(String dsn, SentryClientFactory sentryClientFactory) {
        SentryClient sentryClient = SentryClientFactory.sentryClient(dsn, sentryClientFactory);
        setStoredClient(sentryClient);
        return sentryClient;
    }
```

**Review by komamitsu — APPROVED** [review]
👍

**Comment by komamitsu** [comment]
And you can use latest version 1.6.x.

File: src/main/java/com/treasuredata/bigdam/log/Log.java (hunk 1 above)
Comment (tagomoris) [reply src/main/java/com/treasuredata/bigdam/log/Log.java:99]: `setupSentry` should be called at just once in processes in normal processes, or twice or more in tests. For both cases, implicit singleton objects may cause troubles.

**Comment by tagomoris** [comment]
Ah, right. v1.6.3 looks the latest.

**Comment by tagomoris** [comment]
I pushed updated commits.

**Comment by tagomoris** [comment]
We need to update bigdam-pool, bigdam-queue, bigdam-dddb and bigdam-worker later.

**Comment by komamitsu** [comment]
Okay. I'll take over the updates.

**Pull request merged**